src/Lykke.Service.Zcash.Api.Core/Domain/Operations/IOperation.cs

```diff
@@ -16,6 +16,7 @@ public interface IOperation
         DateTime? DeletedUtc { get; }
         string Hash { get; }
         string Error { get; }
+        BlockchainException.ErrorCode? ErrorCode { get; }
```

Comment:
You are right, here it could be non-nullable. But why do you want to deserialize null to enum (do you mean cast nullable to enum without check)?
[De]serializing of the host object works well, null here means "we have no error", but not "error is unknown" if we had default value.